examples/onboard-aws-rds-aurora-mysql-kinesis/main.tf

```diff
@@ -37,7 +37,7 @@ provider "dsfhub" {
 ################################################################################
 # 1. AWS cloud account
 module "aws-default-account-asset" {
-  source = "../../modules/dsfhub-aws-cloud-account"
```

Comment:
I think we should leave these paths (and probably correct the existing ones) as relative file paths so that we know the automated validation is checking things against our local changes, and not just downloading working copies from the registry. What do you think?